style.css

```diff
@@ -484,3 +503,331 @@ textarea {
   margin-right: auto;
   border-radius: 10px;
 }
+
+@media (min-width: 768px) {
+  body {
+    width: 768px;
+  }
```

Comment:
- [ ] It would be better not to fix the width of the page so your page fits all the desktop screen sizes. I would recommend using `width: 100%` so your page is responsive.

- As you see on the screenshot, the width of your page doesn't fit the screen size which makes your page scroll horizontally.

<img width="958" alt="image" src="https://user-images.githubusercontent.com/77190203/161114558-a352ec9a-57ff-442f-9d33-dfe093a45934.png">